<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="listWidget_purchase_list">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_refresh_purchase_list">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit_name">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_quantity">
          <property name="placeholderText">
           <string>Quantity</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_unit_of_measurement">
          <property name="placeholderText">
           <string>Unit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_purchase_date">
          <property name="placeholderText">
           <string>Date(yyyy/mm/dd)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_add_new_position">
          <property name="text">
           <string>Add position</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit_set_quantity">
          <property name="placeholderText">
           <string>New quantity</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_set_quantity">
          <property name="text">
           <string>&lt;-- Set</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_set_purchase_date">
          <property name="placeholderText">
           <string>New date(yyyy/mm/dd)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_set_purchase_date">
          <property name="text">
           <string>&lt;-- Set</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
